packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -451,6 +451,12 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with AutomaticK
     _renderEditable.handleTapDown(details);
   }
 
+  void _handleForcePressStarted(ForcePressDetails details) {
+    _renderEditable.setLastTapDownPosition(details.globalPosition);
+    // The cause is not technically double tap, but we would like the same thing to happen.
+    _renderEditable.selectWord(cause: SelectionChangedCause.doubleTap);
```

Comment:
I feel the pain. Please open an issue describing the need to decouple cause and effect with this enum. I think there might be some subtle differences in whether it shows a toolbar or not. Do check first.

Reply:
ya, let's leave this